blockly/src/client/Client.java

```diff
@@ -142,13 +142,7 @@ private static void createHero() {
   }
 
   private static void createSystems() {
-    Game.add(new CollisionSystem());
-    Game.add(new AISystem());
-    Game.add(new HealthSystem());
-    Game.add(new PathSystem());
-    Game.add(new ProjectileSystem());
-    Game.add(new HealthBarSystem());
-    Game.add(new HudSystem());
```

Comment:
Hier fehlt das `SpikeSystem` und das `IdleSoundSystem`.

Ist das beabsichtigt? Wenn nicht, dann sollte ich die Änderung rückgängig machen.

Mit `Game.add(StandardSystems.standardSystems());` füge ich diese aber auch hinzu.

Reply:
Das ist genau der Punkt, den ich mit 
> Es ist wichtig, die Freiheit zu haben, diese Systeme je nach Bedarf für das Spiel auszuwählen.
in #1587 meinte. 
Man braucht eine Lösung, wo man essentielle System laden kann, aber trotzdem eine Auswahl treffen darf. Zum Beispiel muss ich für mein DevDungeon das `HealthSystem` mit einer eigene Klasse ersetzen (für eigene Logik). Ich glaube das braucht eine grundlegenden anderen Ansatz.

Reply:
@tgrothe Die Diskussion ist noch nicht mal fertig, und Du rennst schon wieder mit dem PR los?! Das erzeugt nur wieder unnötig Arbeit. Ich setzte das Ding auf "Draft", bis die Diskussion abgeschlossen ist und eine Lösung gefunden wurde.

Reply:
@tgrothe Ich habe den [ersten Kommentar](https://github.com/Dungeon-CampusMinden/Dungeon/issues/1587#issue-2423105217) in #1587 mal angepasst und um die (aus meiner Sicht) logischen nächsten Schritte ergänzt. Da ist noch einiges an konzeptioneller Arbeit zu erledigen, bevor ich an eine Implementierung denken würde. Du darfst gern mit nachdenken und Vorschläge machen :)

Reply:
@cagix Alles klar, dann geht es zunächst dort weiter. :)